KiedyKolos/Common/KiedyKolos.Core/Validators/Subject/CreateYearCourseSubjectCommandValidator.cs

```diff
@@ -0,0 +1,31 @@
+using FluentValidation;
+using KiedyKolos.Core.Commands;
+using KiedyKolos.Core.Interfaces;
+
+namespace KiedyKolos.Core.Validators.Subject
+{
+    public class CreateYearCourseSubjectCommandValidator : AbstractValidator<CreateYearCourseSubjectCommand>
```

Comment:
CreateYearCourseSubjectCommandValidator should check, if such subject doesn't already exist(Name and ShortName should both be unique for one year course).